libtrellis/src/Bitstream.cpp

```diff
@@ -147,7 +151,7 @@ class BitstreamReadWriter {
 
     // Decompress and copy multiple bytes into an OutputIterator and update CRC
     template<typename T>
-    void get_compressed_bytes(T out, size_t count, array<uint8_t, 8> compression_dict) {
+    void get_compressed_bytes(T out, size_t count, array<uint8_t, 16> compression_dict) {
```

Comment:
this all looks nicely done, but for piece of mind can you double check a compressed bitstream still works on an ecp5 with these changes?

Reply:
Did this while working on PR, but will do it again.

Reply:
Checked right with couple of bitstreams now on ULX3S board, all fine.